cmake/configs/default.cmake

```diff
@@ -347,3 +347,4 @@ hunter_config(zookeeper VERSION 3.4.9-p2)
 hunter_config(tacopie VERSION 2.4.0-h1)
 hunter_config(cpp_redis VERSION 3.5.0-h1)
 hunter_config(IF97 VERSION 2.1.2)
+hunter_config(cpr VERSION 1.3.0)
```

Comment:
Try to keep it alphabetical.

Reply:
I think this ship has long sailed

Reply:
Too bad there is no automatic check. Though I believe it's possible to craft one :)